docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -64,6 +64,141 @@ The top bar of Teleport Connect consists of:
+directory containing the tsh binary, even if [tsh is not globally available](#using-tsh-outside-of-teleport-connect).
+
+When using [trusted clusters](../management/admin/trustedclusters.mdx), the cluster selector allows
+you to determine which cluster the shell session will be bound to. The selected cluster will be
+reflected in both the tab title and the status bar.
+
+## Connecting to a Kubernetes cluster
+
+1. Open a tab with cluster resources by clicking on the plus symbol at the right end of the tab bar.
+   You can also press <span style="white-space: nowrap;">`Ctrl/Cmd + T`</span> to achieve the same result.
+1. Look for the cluster you wish to connect to and click the Connect button.
+
+Alternatively, you can look for the cluster in the search bar and press `Enter`
+to connect to it.
+
+A new local terminal tab will open which is preconfigured with the `KUBECONFIG`
+environment variable pointing to a configuration for the specified cluster. Any
+tools that respect the `KUBECONFIG` environment variable (`kubectl`, `helm`,
+etc.) will work without additional configuration. To identify the path to this
+config for use in other tools, inspect the value of the `KUBECONFIG`
+environment variable.
+
+The value of the `KUBECONFIG` environment variable will persist between app
+restarts, but this configuration file will not be available or usable when the
+Kubernetes cluster is removed from the connection list in the top left, or when
+Teleport Connect is closed.
+
+## Connecting to a database
+
+1. Open a tab with cluster resources by clicking on the plus symbol at the end of the tab bar. You
+  can also press <span style="white-space: nowrap;">`Ctrl/Cmd + T`</span> to achieve the same result.
+1. Look for the database server you wish to connect to and click the Connect button.
+1. Select or enter the database user you wish to use and press `Enter`.
+
+A new tab will open with a new connection established between your device and the database server.
+
+Alternatively, you can look for the database in the search bar and press `Enter` to connect to it.
+
+This connection will remain active until you click the Close Connection button or close Teleport
+Connect. The port number will persist between app restarts—you can set up your favorite client
+without worrying about the port suddenly changing.
+
+### With a GUI client
+
+To connect with a GUI client, follow the instructions in the database connection tab under the
+Connect with GUI section.
+
+### With a CLI client
+
+The database connection tab shows the command that can be used to connect to the database. You can
+modify the database name of the connection and then click the Run button to open a new terminal tab
+with that command executed.
+
+## Connecting to an application
+
+Teleport Connect supports launching applications in the browser, as well as creating
+authenticated tunnels for HTTP and TCP applications.
```

Comment:
I think we shouldn't use the 'HTTP apps' term. In other docs we say 'Web apps' https://goteleport.com/docs/application-access/guides/connecting-apps/.